src/R/Components/Test/Repl/ReplCommandTest.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition.Hosting;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.R.Components.ContentTypes;
+using Microsoft.R.Components.Controller;
+using Microsoft.R.Components.InteractiveWorkflow;
+using Microsoft.R.Host.Client;
+using Microsoft.R.Host.Client.Test;
+using Microsoft.R.Support.Settings;
+using Microsoft.UnitTests.Core.Threading;
+using Microsoft.UnitTests.Core.XUnit;
+using Microsoft.VisualStudio.Editor.Mocks;
+using Microsoft.VisualStudio.R.Package.Repl.Commands;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.R.Components.Test.Repl {
+    [ExcludeFromCodeCoverage]
+    [Collection(CollectionNames.NonParallel)]
```

Comment:
You can remove this - `RComponentsMefCatalogFixture` creates unique MEF container for every test, so they can run in parallel